help/en/html/hardware/marklin/index.shtml

```diff
@@ -63,7 +68,33 @@
       <img src="images/MarklinCS2SendCommand.png" width="348" height="128" alt=
       "Send Packet Tool"><br>
       Enter a message as a hex string, starting with the prefix 0x e.g. <code>0x4467</code>. Press
-      [Send] and wait for a reply from the CS2 connection to appear in the Reply field. 
+      [Send] and wait for a reply from the CS2 connection to appear in the Reply field.
+
+
+      <h2 id="cdb">CdB CC-Schnitte 2</h2>
+
+      <p><span class="since">Since <a href="jmri5.3.5.shtml">JMRI 5.3.5</a></span>
+      JMRI can connect to a M&auml;rklin Command Station 2 using a USB interface.</p>
```

Comment:
@ralflang 
Is this correct? Or is CC-Schnitte 2 only supposed to work with the MobileStation 2?